panels/user-accounts/um-photo-dialog.c

```diff
@@ -375,13 +375,39 @@ menu_item_for_filename (UmPhotoDialog *um,
         return menuitem;
 }
 
+static GSList *
+get_facesdirs ()
+{
+        GSList *facesdirs = NULL;
+        const gchar * const * data_dirs;
+        int i;
+        const char *override_facesdir;
+
+        override_facesdir = g_getenv ("CC_USER_ACCOUNTS_PANEL_FACESDIR");
+        if (override_facesdir == NULL || !g_strcmp0 (override_facesdir, "")) {
+                override_facesdir = USER_ACCOUNTS_PANEL_FACESDIR;
```

Comment:
I would make these additive instead of mutually exclusive. You can just append both to the lookup list (the env var should take precedence)